api/descriptions/collections/name_delete.md

```diff
@@ -1,11 +1,24 @@
-Deletes a collection from the system.
+Deletes a collection.
 
-The following example curl command deletes a collection named *my collection*.
-Because spaces are considered [unsafe characters in a URL](https://www.ietf.org/rfc/rfc1738.txt), they must be encoded with the value `%20`.
+To invoke this endpoint in the Console UI:
+
+1. Navigate to **Manage > Collections and Tags > Collections**.
+2. Click the dotted icon under the **Actions** column to open up the menu options. **Note:** The default collections do not have a dotted icon in the **Actions** column.
+3. Click the **Delete** button to initiate the deletion. 
+4. Click the **Delete Collection** button to confirm the deletion.
+
+### cURL Request
```

Comment:
@KICKdesigns Let's explicitly define what values can be passed to the URL param `{id}`. In this case, it's the collection name. Collection names can be retrieved from the `name` key in the GET /api/v1/collections endpoint.